.github/workflows/SDSGAS-CI.yaml

```diff
@@ -0,0 +1,23 @@
+name: SDSGAS-CI
+on:
+  push:
+    branches: main
+  pull_request:
+    branches: main
+jobs:
+  build:
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+      - name: Use Node.js v18
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - name: Install Dependencies
+        run: npm ci
+      - name: Run linters
```

Comment:
Since this isn't actually linting, what if we renamed it to something like "verify format"?

Reply:
Updated it to 'Verify Format'